docs/locale/fr/LC_MESSAGES/adminguide/index.po

```diff
@@ -0,0 +1,24 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2016-present, Scille SAS
+# This file is distributed under the same license as the Parsec package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
```

Comment:
This does not come from a template? How about putting something like these in all files:
```suggestion
# Copyright (C) 2016-present, Scille SAS
# This file is distributed under the same license as the Parsec package.
# Maintained by the parsec-cloud team. 
# https://parsec.cloud/
```

Reply:
done